ocs_ci/deployment/disconnected.py

```
@@ -37,6 +37,45 @@
+        configure_registries (bool): Whether to configure /etc/containers/registries.conf for internal images
+
+    Returns:
+        str: mirrored catalog image URL
+
+    """
+    logger.info(f"Mirroring FDF catalog: {catalog_image}")
+
+    # Set mirror_registry in config if provided
+    if mirror_registry:
+        config.DEPLOYMENT["mirror_registry"] = mirror_registry
+        logger.info(f"Set mirror_registry in config: {mirror_registry}")
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)